CSVConvert.py

```diff
@@ -393,24 +395,32 @@ def process_mapping(line, test=False):
         return value, elems
     return line, None
 
+
 def read_mapping_template(mapping_path):
     """Given a path to a mapping template file, read the lines and
     return them as an array."""
     template_lines = []
     try:
         with open(mapping_path, 'r') as f:
-            lines = f.readlines()
+            lines = csv.reader(f)
             for line in lines:
-                if line.startswith("#"):
+                if len(line) == 0:
```

Comment:
just to be sure that this picks up a whitespace-only line, make this `len(line.strip())`?

Reply:
agreed, good catch, will change to that

Reply:
I actually changed it in the for loop below where I parse each comma separated value